addons/packages/kapp-controller/0.30.0/bundle/config/overlays/update-deployment.yaml

```diff
@@ -29,15 +29,42 @@ spec:
           #@overlay/match by="name"
           - name: KAPPCTRL_API_PORT
             value: #@ str(values.kappController.deployment.apiPort)
+        #@ if values.kappController.deployment.hostNetwork:
```

Comment:
Since this templating work is for injecting coreDNSIP lets use that for the condition instead of hostNetwork.

Reply:
Changed